docs/modules/kafka.md

```diff
@@ -1,10 +1,12 @@
-# Kafka (KRaft)
+# Kafka
 
 Since <a href="https://github.com/testcontainers/testcontainers-go/releases/tag/v0.24.0"><span class="tc-version">:material-tag: v0.24.0</span></a>
 
 ## Introduction
 
-The Testcontainers module for KRaft: [Apache Kafka Without ZooKeeper](https://developer.confluent.io/learn/kraft).
+The Testcontainers module for Kafka.
+
+This module would run Kafka in Kraft mode: [Apache Kafka Without ZooKeeper](https://developer.confluent.io/learn/kraft/) and it supports both [Apache Kafka](https://kafka.apache.org/) and [Confluent](https://docs.confluent.io/kafka/overview.html) images.
```

Comment:
It can be helpful for the users if we notice here (an outcome of https://github.com/testcontainers/testcontainers-go/pull/3249#discussion_r2507351306):

1. Confluent images doesn't support (stably) graceful shutdown.
2. Confluent images are considered legacy (supported for backward compatibility) and Apache Kafka images are recommended.